src/main/scala/org/apache/spark/sql/delta/Checkpoints.scala

```diff
@@ -135,10 +135,10 @@ trait Checkpoints extends DeltaLogging {
   /** Loads the checkpoint metadata from the _last_checkpoint file. */
   private def loadMetadataFromFile(tries: Int): Option[CheckpointMetaData] = {
     try {
-      val checkpointMetaData = store.read(LAST_CHECKPOINT)
-      val checkpointMetadata =
-        JsonUtils.mapper.readValue[CheckpointMetaData](checkpointMetaData.head)
-      Some(checkpointMetadata)
+      val lastCheckpointMetaData = store.read(LAST_CHECKPOINT)
```

Comment:
We usually use `Metadata` (that is, metadata is a single word). 

And the real difference between the two similarly named variables is that one is the json and the other is the actual metadata. So you can name them `checkpointMetadataJson` and `checkpointMetadata`.

Reply:
done.